plugins/patterns/patterns_renderer.rb

```diff
@@ -23,10 +23,13 @@ def content(description)
       puts "There are no patterns or tasks."
     end
 
-    if description.packages && description.packages.package_system == "dpkg"
+    if description.patterns.patterns_system == "tasksel"
       puts "Note: Tasks on Debian-like systems are treated as patterns."
     end
 
+    puts "Pattern Manager: #{description.patterns.patterns_system}" unless \
+      description.patterns.empty?
```

Comment:
The "\" should not be needed, but personally I would prefer to put the if statement on top if it does not fit the line. But I am not sure about the style guide.